lib/declare_schema/model/foreign_key_definition.rb

```diff
@@ -12,7 +12,7 @@ class ForeignKeyDefinition
 
       def initialize(model, foreign_key, **options)
         @model = model
-        @foreign_key = foreign_key.to_s.presence
+        @foreign_key = foreign_key.to_s.presence or raise ArgumentError "Foreign key cannot be empty: #{foreign_key.inspect}"
```

Comment:
How about "must not be empty"? I recall that being preferred grammatically.

Reply:
Yes, using "must not" makes it sound like more of an imperative. I'll change the wording.